src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -786,19 +786,6 @@ public void parseWarnsAboutUnmatchedContentInEntry() throws IOException {
     }
 
     @Test
-    @Ignore("Ignoring because this is an edge case")
```

Comment:
JabRef returns one entry, which pretty good. I would like to have this checked in tests if we adress it.

The test has to be adapted accordingly.

Result is 

`@article{test,
  author = {author bracket #too##much#}
}`

Would it be OK if you include this as expected case, even if the `}` gets lost...

This documents the drawback of JabRef.

OK, additional documentation at https://help.jabref.org/en/Bibtex would be better, but none of us has time for that.